{{ cookiecutter.__project_name_kebab }}/.github/workflows/publish.yml

```diff
@@ -21,7 +22,7 @@ jobs:
       - name: Set up Python 3.8
```

Comment:
```suggestion
```

Reply:
god I need to step away from the computer